### ISSUE TITLE
Feature: Add avatarURL to the DB

### ISSUE THREAD

**Issue by bdougie** [open-sauced/actions#22]
### Type of feature

Feature

### Current behavior

Currently, we are story-only contributor login info thanks to #21. We need to add Avatar URLs as well. This data is already being fetched from the onegraph endpoints. 

### Desired solution

```
contributors: [{login: "bdougie", avatarURL: "https://avatars.githubusercontent.com/u/5713670?v=4"]
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bdougie** [comment]
note about arrays in supabase - https://supabase.com/docs/guides/database/arrays

**Comment by bdougie** [comment]
postgres arrays will not work with this. We will need to use JSON, this will also need to be new column in order to not break UI.

**Comment by 0-vortex** [comment]
is this still needed?

**Comment by 0-vortex** [comment]
This got partially fixed with #36, nonetheless, that information is accessible through the database now and we can start to migrate some of our scripts over to backend requests, removing script complexity in favour of webhooks like events 🍕
